zebra-utils/Cargo.toml

```diff
@@ -76,6 +88,7 @@ thiserror = "1.0.48"
 
 zebra-node-services = { path = "../zebra-node-services", version = "1.0.0-beta.32" }
 zebra-chain = { path = "../zebra-chain", version = "1.0.0-beta.32" }
+zebra-scan = { path = "../zebra-scan", version="0.1.0-alpha.1",optional = true }
```

Comment:
Optional Nitpick: spacing

```suggestion
zebra-scan = { path = "../zebra-scan", version = "0.1.0-alpha.1", optional = true }
```

Reply:
I formatted the file manually because I didn't have any tool for it. I tried to imitate the formatting based on existing parts. Did you notice this bug just by looking at the file, or did you use some tool?

Reply:
Just noticed it by reading, I'm not sure if there's a lint tool. But here is the convention:
https://doc.rust-lang.org/nightly/style-guide/cargo.html

Reply:
Thanks for sharing those.